source/platforms/databricks.md

```diff
@@ -66,6 +88,24 @@ gdf
 
 ```
 
+Connect to the dask client and submit tasks.
+
+```python
+from dask.distributed import Client
+from dask_databricks import DatabricksCluster
+
+cluster = DatabricksCluster()
+client = Client(cluster)
```

Comment:
I think we should use the simplified `get_client` version here to reduce information overload.

```suggestion
import dask_databricks

client = dask_databricks.get_client()
```